style.css

```diff
@@ -334,3 +334,112 @@ body {
   margin-right: 10px;
   margin-left: 10px;
 }
+
+.contact_form {
+  margin-top: 24px;
+  display: flex;
+  border-radius: 50px 0 0;
+  flex-direction: column;
+  background: #6070ff;
+  background-image: url("logos/form-shapes-mobile.png");
+  background-repeat: inherit;
+  background-size: 375px 650px;
+  height: 779px;
+}
+
+.contact_form h2 {
+  margin-top: 103px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  font-weight: 700, bold;
+  color: white;
+  margin-bottom: 12px;
+  text-align: center;
+}
+
+.contact_form p {
+  font-family: 'Poppins', sans-serif;
+  color: white;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 70px;
+}
+
+#full-name,
+#email-address {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  height: 48px;
+  margin-bottom: 12px;
+}
+
+#message {
+  border: 1px solid #cfd8dc;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  border-radius: 8px;
+  height: 170px;
+  margin-bottom: 12px;
+}
+
+#fs-frm ul {
+  list-style: none;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+#fs-frm li {
+  display: flex;
+  flex-direction: column;
+  margin: 0.5em 0;
+}
+
+.Sbtn {
```

Comment:
- [ ] The text is not centered, please make it center and try to duplicate the Figma design as much as possible